src/test/java/com/daml/extensions/testing/ledger/SandboxManagerIT.java

```diff
@@ -40,7 +41,7 @@ public void managerStopsSandboxGracefully() throws Exception {
             Duration.ofMinutes(1),
             Duration.ofSeconds(10),
             new String[0],
-            DAR_PATH,
+            new Path[] { DAR_PATH },
```

Comment:
I think that if we had a test with multiple DAR-s then it would fail because as far as I know `daml script` does not like multiple `--dar` options.

Reply:
Thanks a lot for this Gabor ... did not have time test that and assumed it... will not assume that again ...